05-onchain.md

```diff
@@ -342,21 +382,24 @@ to a convenient address.
 For any committed HTLC that does not have an output in this
 commitment transaction, the node MUST fail the corresponding incoming
 HTLC (if any) once the commitment transaction has reached reasonable
-depth, and MAY fail it sooner if no valid commitment transaction
+depth, and MAY fail it sooner if no *valid* commitment transaction
 contains an output corresponding to the HTLC.
 
-## Rationale
+### Rationale
 
-If the commitment transaction is theirs, the only way to spend the
+If the commitment transaction is *remotes*, the only way to spend the
```

Comment:
*remote's* or perhaps clearer, "belongs to the remote node"

Reply:
fixed